OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1096,13 +1142,36 @@ void testExpressionBasedBushingForceRotational()
         ASSERT_EQUAL(analytical_moment, model_forces[4], 2e-4);
     }
 
-    osimModel.disownAllComponents();
-
     manager.getStateStorage().print("expression_based_bushing_rotational_model_states.sto");
 
     // Save the forces
     reporter->getForceStorage().print("expression_based_bushing_rotational_model_forces.mot");
 
+    // Now add damping to the bushing force and make sure energy is monotonically decreasing
```

Comment:
Could dump this code into a lambda since it's repeated 3 times.

Reply:
True 👍

Reply:
Due to scope issues (function has to be called outside block) I used a separate utility function to avoid duplication. Please let me know what you think.

Reply:
LGTM 🐴